corehq/apps/app_manager/models.py

```diff
@@ -6619,6 +6655,116 @@ def save(self, *args, **kwargs):
         super(GlobalAppConfig, self).save(*args, **kwargs)
 
 
+class Translation(models.Model):
```

Comment:
We talked about this, but leaving as a comment for other reviewers: I think it'll be far more maintainable to have this structure be a flat list, where keys are id strings, rather than a structure that mimics modules/forms.

Reply:
...and doing this would have make enough changes in this PR that I'm not going to review much more.